### ISSUE TITLE
Browser dark theme interferes with editor text color

### ISSUE THREAD

**Issue by jamov** [dasmikko/strapi-tiptap-editor#13]
Hey,

Looks like browser's dark theme is interfering with the editor styles.
Below are text color styles which browsers are using in different cases.

Strapi and Tiptap versions
```
"@strapi/strapi": "4.1.7",
"strapi-tiptap-editor": "0.7.4",
```
Tested on Windows with Chrome, Firefox, and Edge

Browser & Strapi in dark mode = dark editor text on dark background)
```
@media (prefers-color-scheme: dark)
<style>
.bGOtGv .ProseMirror {
    color: rgb(24, 24, 38);
}
```

Browser in light mode & Strapi in dark mode works correctly = light editor text on dark background
```
@media (prefers-color-scheme: light)
.bGOtGv .ProseMirror {
  color: rgb(255, 255, 255);
}
```

**Comment by dasmikko** [comment]
Hi!

Thanks for reporting the bug. I have fixed this issue in 0.7.5, so please upgrade to that version.

**Comment by jamov** [comment]
Thanks for fixing it so quickly! 👍
